.pipelines/templates/e2e-test-azure.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
         steps:
         - task: GoTool@0
           inputs:
-            version: 1.16
+            version: 1.17
```

Comment:
Should we just remove this, since we are moving to staging pool with https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/621?

Reply:
Yeah, we will remove it as part of #621. Until that PR is merged, we need the version pinned to 1.17.